adoc/deployment-bootstrap.adoc

```diff
@@ -376,6 +376,15 @@ For clusters provisioned by any other method, ensure virtual machines are set to
 For more information, refer to: link:https://docs.vmware.com/en/VMware-vSphere/6.7/Cloud-Native-Storage/GUID-3501C3F2-7D7C-45E9-B20A-F3F70D1E4679.html[Configure Kubernetes Cluster Virtual Machines] .
 ====
 
+[IMPORTANT]
+====
+All virtual machines must exist in a folder and provide as the `folder` variable in the `vsphere.conf` before bootstrap.
+
+Clusters provisioned following link:{docurl}single-html/caasp-deployment/#_using_terraform[Deploying VMs from the Template] with `cpi_enable = true` automatically create and place all cluster node virtual machines inside a `*-cluster` folder.
+
```

Comment:
What is I do NOT want to use a folder and want to have all VMs in the "root"? What is the default I need to enter for "folder"?

Reply:
It is required by the in-tree plugin to look for a folder under `/vm`. Storage class cannot be used with an invalid `folder` variable.